test_times.py

```diff
@@ -0,0 +1,34 @@
+from times import *
+import pytest
+
+def test_given_input():
+    large0 = time_range("2010-01-12 10:00:00", "2010-01-12 12:00:00")
+    short0 = time_range("2010-01-12 10:30:00", "2010-01-12 10:45:00", 2, 60)
+    result0 = compute_overlap_time(large0, short0)
+    expected0 = [('2010-01-12 10:30:00', '2010-01-12 10:37:00'), ('2010-01-12 10:38:00', '2010-01-12 10:45:00')]
+    assert result0 == expected0
+    #Two time ranges that do not overlap
+def test_not_overlapping():
```

Comment:
I like that you've documented each test with a comment!

This is just style, but a typical Python convention is that there should be an empty line before a function definition, and the documentation goes after, with an indentation!
```suggestion

def test_not_overlapping():
    """ Test two time ranges that don't overlap."""
```
We'll talk more about documentation in the next class!

(same for other tests)